ci-operator/step-registry/etcd-encryption/etcd-encryption-commands.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
```

Comment:
In prow ci, to be more restrict for shell syntax, generally we are setting, do you mind to follow the same rule.
```
set -o errexit
set -o nounset
set -o pipefail
```